test/vcf/record_cache_test.cpp

```diff
@@ -126,5 +139,11 @@ namespace ebi
             // the first occurrence should not be reported again
             CHECK( count_duplicates_and_rethrow_error(cache, {100, "GA", {"GT"}}) == 1 );
         }
+        SECTION("Symbolic Duplicates")
+        {
+            CHECK( count_duplicates_and_rethrow_error(cache, {107, "C", {"<INS>"}}) == 0 );
+            // This won't throw an error. Only warning would be generated, which isn't detected here.
+            CHECK( count_duplicates_and_rethrow_error(cache, {107, "C", {"<INS>"}}) == 0 );
```

Comment:
These tests are incomplete because the function `count_duplicates_and_rethrow_error` is using only `get_duplicates` and not `get_symbolic_duplicates`. The comment above is misleading because the code flow does not involve warnings.

Another `TEST_CASE` should be added that uses `get_symbolic_duplicates` and test that with both symbolic and regular variants.